data/duke.txt

```diff
@@ -0,0 +1,4 @@
+[D][X] do ip (by: Oct 12 2021, 7:12 PM)
```

Comment:
This file doesn't need to be revision-controlled.

Reply:
Probably not.